package.json

```diff
@@ -318,6 +326,20 @@
                                                         ],
                                                         "default": "String"
                                                     },
+                                                    "mode": {
+                                                        "type": "string",
```

Comment:
I'm just hoping here that omitting this will result in `None` on the Rust side...